src/function/list/list_reduce.cpp

```diff
@@ -22,12 +22,28 @@ static std::unique_ptr<FunctionBindData> bindFunc(ScalarBindFuncInput input) {
         ListType::getChildType(input.arguments[0]->getDataType()).copy());
 }
 
+static uint32_t findIndex(const std::vector<std::string>& strings, const std::string& str) {
```

Comment:
use common::idx_t instead of uint32_t. Double check other places in this PR as well.